### ISSUE TITLE
Incompatible w/ Seamonkey (was: Mark as installable in Seamonkey)

### ISSUE THREAD

**Issue by pde** [EFForg/privacybadgerfirefox#126]
Not sure if there are any gotchas for this due to Jetpack

**Comment by cooperq** [comment]
I have reopened it and will test privacy badger on seamonkey

**Comment by cooperq** [comment]
@mylittleplaceholder is also interested in this.

**Comment by Woi** [comment]
Please add my vote, too.

**Comment by rodneyrod** [comment]
I'd be interested in testing any plugins as they come out.

**Comment by zoff99** [comment]
+1 please at least mark it compatible that seamonkey users can install and test it.
does not really make any sense to NOT mark it compatible

**Comment by cooperq** [comment]
If someone wants to submit a patch marking this as compatible with seamonkey I will happily merge it.

**Comment by zoff99** [comment]
just add these lines to install.rdf
<br><br>
<pre>
	
   <em:targetApplication>
      <Description>
        <em:id>{92650c4d-4b8e-4d2a-b7eb-24ecf4f6b63a}</em:id>
        <em:minVersion>1.1</em:minVersion>
        <em:maxVersion>2.*</em:maxVersion>
      </Description>
   </em:targetApplication>
</pre>

**Comment by zoff99** [comment]
at this position just under the firefox part
<br>
<br>
![unbenannt](https://cloud.githubusercontent.com/assets/6833516/12865464/80291946-ccac-11e5-8ab1-fade30778fc2.PNG)

**Comment by zoff99** [comment]
any news on this?

**Comment by teicher** [comment]
Well, according to Add-Ons directory, it should be compatible, but when I download and run the xpi,
it just sais "...count not be installed because it is incompatibe with Seamonkey 2.39".
Since Ghostery dropped Seamonkey Support lately, Seamonkey users desperately need some working privacy tool for current versions...

**Comment by zoff99** [comment]
please just add those lines to install.rdf

**Comment by cooperq** [comment]
I did! and I marked the version in AMO as compatible with seamonkey... can you try downloading the sefl hosted version from https://eff.org/privacybadger
can you also unpack the version from AMO and tell me whats in the install.rdf

**Comment by zoff99** [comment]
does NOT work. i downloaded from this link now:
https://www.eff.org/files/privacy-badger-latest.xpi

and install.rdf does not contain the lines


[install.rdf___.txt](https://github.com/EFForg/privacybadgerfirefox/files/219639/install.rdf___.txt)

**Comment by zoff99** [comment]
what is AMO??

**Comment by cooperq** [comment]
AMO is addons.mozilla.org, privacy badger can be downloaded here: https://addons.mozilla.org/en-US/firefox/addon/privacy-badger-firefox/

I will have to look into why the self hosted version didn't get the changes that I made.

**Comment by zoff99** [comment]
AMO also does NOT work.
install.rdf does not contain seamonkey entry!

**Comment by cooperq** [comment]
Weird, I makred it as compatible with seamonkey on AMO. I will look into this further.

**Comment by cooperq** [comment]
Here is the listing on AMO, which says it's compatible with seamonkey. I don't really know what else I can do. 
![pb-seamonkey](https://cloud.githubusercontent.com/assets/64981/14749723/01b5d7f6-0876-11e6-989c-b1353f64c212.png)

**Comment by zoff99** [comment]
you need to make sure that entry is reall inside the install.rdf file inside the xpi file
you MUST put that entry there yourself.
only marking it does not do anything

**Comment by antdude** [comment]
FYI. It is still not installable in v2.40. :(

**Comment by zoff99** [comment]
does PB have a menu item or a config screen?
when i manually make it compatible with seamonkey, i can not see if it's working or not.
there is no icons, no menu, no indication ..

**Comment by cooperq** [comment]
Yes there should be a badger icon that you can click on to see a list of all the trackers being blocked.

**Comment by zoff99** [comment]
on seamonkey they icon is not there.
can you add an item to a normal menu somewhere?

**Comment by cooperq** [comment]
If someone wants to submit a pull request adding an item to a normal menu they are welcome to. I don't personally have time to work on it.
